### PR TITLE
Separated AtomConstants to different files, edited the code to use th…

### PR DESCRIPTION
**Pull request by ElJerry**
…e new separated constants.

### CODE REVIEW

File: lab/src/Microsoft.SyndicationFeed/src/Atom/AtomLinkTypes.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.SyndicationFeed.Atom
+{
+    public static class AtomLinkTypes
+    {
+        public const string Alternate = "alternate";
+        public const string Relative = "rel";
```
Comment (drago-draganov) [inline lab/src/Microsoft.SyndicationFeed/src/Atom/AtomLinkTypes.cs]: What is this?

File: lab/src/Microsoft.SyndicationFeed/src/Atom/AtomMediaTypes.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.SyndicationFeed.Atom
+{
+    public static class AtomMediaTypes
+    {
+        public const string AtomMediaType = "application/atom+xml";
```
Comment (drago-draganov) [inline lab/src/Microsoft.SyndicationFeed/src/Atom/AtomMediaTypes.cs]: Remove the ending 'MediaType' -  AtomMediaType -> Atom

File: lab/src/Microsoft.SyndicationFeed/src/Atom/AtomMediaTypes.cs [hunk 3]
```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.SyndicationFeed.Atom
+{
+    public static class AtomMediaTypes
```
Comment (drago-draganov) [inline lab/src/Microsoft.SyndicationFeed/src/Atom/AtomMediaTypes.cs]: No need to be public

File: lab/src/Microsoft.SyndicationFeed/src/Atom/AtomElementNames.cs [hunk 4]
```diff
@@ -27,10 +27,10 @@ public static class AtomElementNames
         public const string Label = "label";
         public const string Scheme = "scheme";
         public const string Source = "src";
-        public const string Self = "self";
         public const string Uri = "uri";
         public const string Term = "term";
         public const string Type = "type";
         public const string Length = "length";
+        public const string Rel = "rel";
```
Comment (drago-draganov) [inline lab/src/Microsoft.SyndicationFeed/src/Atom/AtomElementNames.cs]: Rel, Href, Src, Length, Type are not Atom elements. They are attributes.

File: lab/src/Microsoft.SyndicationFeed/src/Atom/AtomAttributes.cs [hunk 5]
```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.SyndicationFeed.Atom
+{
+    public static class AtomAttributes
```
Comment (drago-draganov) [inline lab/src/Microsoft.SyndicationFeed/src/Atom/AtomAttributes.cs]: Why do we need to expose these at first place?

File: lab/src/Microsoft.SyndicationFeed/src/Rss20/Rss20ElementNames.cs [hunk 6]
```diff
@@ -28,5 +28,13 @@ public static class Rss20ElementNames
         public const string Title = "title";
         public const string Url = "url";
         public const string Version = "version";
+
```
Comment (ElJerry) [inline lab/src/Microsoft.SyndicationFeed/src/Rss20/Rss20ElementNames.cs]: The separation is only to show the new elements. We can always merge them with the other ones.

File: lab/src/Microsoft.SyndicationFeed/examples/RssWriteItemWithCustomElementExample.cs [hunk 7]
```diff
@@ -41,7 +47,7 @@ class RssWriteItemWithCustomElement
             var content = new SyndicationContent(formatter.CreateContent(item));
 
             // Add custom fields/attributes
-            content.AddField(new SyndicationContent("CustomElement", "My Value"));
+            content.AddField(new SyndicationContent("example:customElement", "http://contoso.com/syndication/feed/examples", "Custom Value"));
```
Comment (jimmyca15) [inline lab/src/Microsoft.SyndicationFeed/examples/RssWriteItemWithCustomElementExample.cs]: http://contoso.com/syndication/feed/examples should be in a constant

const string EXAMPLE_NS = "http://contoso.com/syndication/feed/examples";

**Comment by drago-draganov** [comment]
Looks good!

**Pull request merged**